README.md

```diff
@@ -70,5 +70,5 @@ fondant --help
 
 There are two options to run the pipeline:
 
-- [Via python files and the Fondant CLI](./src/README.md): how you should run Fondant in production
```

Comment:
This did not point to anything

Reply:
Seems like I deleted it in [this PR](https://github.com/ml6team/fondant-usecase-RAG/pull/40/files#diff-e1d5879ce3e222eb0cdd43051fb8c34c00796467acb091d0b3fa939ee3355cb5). 

It might make sense to re-add it just for the indexing pipeline. WDYT?
If not, I would still add a link for the CLI to the documentation, and keep the link to the notebook.

Reply:
Might be best to go for the second approach since we don't have a file ready to launch the pipeline from (currently organized in a function that creates the pipeline). Updated